sdk/plugin/plugin.go

```diff
@@ -1,24 +1,36 @@
 package plugin
 
 import (
+	"context"
 	"io"
 	"log"
 	"os"
 	"sync"
 
 	"github.com/aserto-dev/ds-load/sdk/common/js"
-	"github.com/aserto-dev/ds-load/sdk/common/msg"
-	"github.com/aserto-dev/ds-load/sdk/transform"
-	"github.com/pkg/errors"
-	"google.golang.org/protobuf/encoding/protojson"
 )
 
+type Fetcher interface {
+	Fetch(ctx context.Context, outputWriter, errorWriter io.Writer) error
+}
+
+type Transform interface {
```

Comment:
please rename interface to Transformer, following the golang naming convention for interfaces https://go.dev/doc/effective_go#interface-names